tedana/selection/select_comps.py

```diff
@@ -80,12 +79,12 @@ def selcomps(seldict, comptable, mmix, manacc, n_echos):
         EXTEND_FACTOR = 2
     RESTRICT_FACTOR = 2
 
-    # List of components
-    midk = []
-    ign = []
+    # Lists of components
     all_comps = np.arange(comptable.shape[0])
-    # acc remains a full list that is whittled down over criteria
-    acc = np.arange(comptable.shape[0])
+    # unclf is a full list that is whittled down over criteria
+    # since the default classification is "accepted", at the end of the tree
+    # the remaining elements in unclf match up to the accepted components
+    unclf = np.arange(comptable.shape[0])
```

Comment:
Why have both `all_comps` and `unclf` defined, here ?

Reply:
They are both referenced independently throughout the function, and `unclf` is edited while `all_comps` is not. Is that kind of what you were asking about?

Reply:
Yes, sorry, I thought it was just confusing that they were both actually defined. It would have made more sense to me if it were something like
```python
all_comps = np.arange(comptable.shape[0])
unclf = all_comps.deepcopy()
```

Reply:
(Realizing "defined" is a really bad term, sorry !)

Reply:
I can do that.